src/lib_json/json_writer.cpp

```diff
@@ -350,7 +350,7 @@ static String valueToQuotedStringN(const char* value, unsigned length,
 }
 
 String valueToQuotedString(const char* value) {
-  return valueToQuotedStringN(value, static_cast<unsigned int>(strlen(value)));
+  return valueToQuotedStringN(value, static_cast<size_t>(strlen(value)));
```

Comment:
strlen already returns size_t. Better to just remove the static_cast

Reply:
fix it.
https://github.com/open-source-parsers/jsoncpp/pull/1257/commits/cd46649236d7e8e15abfb6c7363f1298a95a0c8b